src/IssueViz.Security/Taint/TaintList/TaintVulnerabilitiesControl.xaml

```diff
@@ -1,15 +1,124 @@
 ﻿<UserControl x:Class="SonarLint.VisualStudio.IssueVisualization.Security.Taint.TaintList.TaintVulnerabilitiesControl"
              x:ClassModifier="internal"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:vsCatalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
+             xmlns:vsImaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
+             xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.14.0"
+             DataContext="{Binding ViewModel, RelativeSource={RelativeSource Mode=Self}}"
+             >
     <UserControl.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceDictionary Source="../../SharedUI/SharedResources.xaml" />
             </ResourceDictionary.MergedDictionaries>
+
+            <!-- Extend the shared data grid row style to add the context menu and navigability data bindings -->
+            <Style BasedOn="{StaticResource BaseDataGridRow}"
+                       TargetType="DataGridRow">
+                <Setter Property="Background" Value="{DynamicResource {x:Static vsTheming:TreeViewColors.BackgroundBrushKey}}"/>
+                <Style.Triggers>
+                    <DataTrigger Binding="{Binding TaintIssueViz.Span, Converter={StaticResource SpanToNavigabilityConverter}}" Value="False">
```

Comment:
This is almost identical to derived style in the hotspot control. The only difference is the path to the property in the Binding. I'm not sure what the best way of removing this duplication is, or whether it's a priority at the moment.
If you can suggest a simple way to do it I'll do it now, otherwise I'll add a sub-task to the backlog.

Reply:
Hm, a dependency property might do it, but I'm not sure if we'd want the exact same behavior. Let's wait and see what the final UX looks like.

Reply:
Sounds good. A dependency property would required both view models to use the same property name, which seems a bit fragile given they are currently independent and invalid bindings aren`t detected until runtime.